desktop/desktop/lib/cookie-auth/index.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
```

Comment:
I think so, but I haven't reviewed them all. Some things in desktop lib may not really be lib stuff as well.